tests/syntax_tests/data/printer/expr/expected/apply.res.txt

```diff
@@ -100,40 +100,34 @@ let g = f(
   c,
 )
 
-let g =
-  f(
-    a =>
-      LongModuleName.functionWithAlongNameThatWrapsTheEditorToTheNextLinexxxxxxxxxxxxxxxxxxxxxx(a),
-    b,
-    c,
-    ...
-  )
-
-let g =
-  f(
-    a =>
-      LongModuleName.functionWithAlongNameThatWrapsTheEditorToTheNextLinexxxxxxxxxxxxxxxxxxxxxx(a),
-    b,
-    c =>
-      LongModuleName.functionWithAlongNameThatWrapsTheEditorToTheNextLinexxxxxxxxxxxxxxxxxxxxxx(
-        d,
-        ...
-      ),
-    ...
-  )
-
-let g =
-  f(
-    a =>
-      LongModuleName.functionWithAlongNameThatWrapsTheEditorToTheNextLinexxxxxxxxxxxxxxxxxxxxxx(
-        a,
-        ...
-      ),
-    b,
-    c =>
-      LongModuleName.functionWithAlongNameThatWrapsTheEditorToTheNextLinexxxxxxxxxxxxxxxxxxxxxx(d),
-    ...
-  )
+let g = f(
```

Comment:
These differences likely due to the fact that there's no attribute on the application node anymore, so some attribute logic used for printing does not kick in anymore.